man/gbz80.7

```diff
@@ -782,9 +814,9 @@ and
 bytes away.
 For example:
 .Bd -literal -offset indent
-    JR Label  ; no-op; encoded offset of 0
+JR Label  ; no-op; encoded offset of 0
```

Comment:
Oh, why?

Reply:
The other code samples weren't indented, and with just three lines, "indented unindentedLabel indented" looked odd to me.

Reply:
I think it helps making the label stand out from the instructions, especially in the absence of syntax highlighting.

If you think the other samples should be indented, then I'll agree.